app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,74 @@
+<div class = "ui grid container">
+  <div class = "row">
+    <div class = "left aligned six wide column">
+      <label for ="entries">Number of entries</label>
+      <select id = "entries" class="ui dropdown">
+        <option value="10">10</option>
+        <option value="25">25</option>
+        <option value="50">50</option>
+        <option value="100">100</option>
+      </select>
+    </div>
+    <div class= "right floated right aligned six wide column">
```

Comment:
@niranjan94  My bad, I thought UI was needed , not the actual searching facility.